meshnet/server.go

```diff
@@ -319,11 +318,10 @@ func (s *Server) DisableMeshnet(context.Context, *pb.Empty) (*pb.MeshnetResponse
 	}
 
 	// stop networker only if mesh peer connected before
-	if s.isPeerConnected {
+	if strings.HasSuffix(s.netw.LastServerName(), ".nord") {
```

Comment:
I'm not a fan of this because Meshnet package shouldn't know about the VPN server naming conventions (which also might change, who knows). 
Suggestion: we should still keep information about being connected to a peer, but instead of it being just a boolean which doesn't tell us much we could keep the peer's hostname. Then in this condition we could check whether we are still connected to that hostname. It's still a bit bad because it relies on "LastServerName" returning the hostname and nothing else, but at least we would be doing checks only with meshnet related information. 

Also, the comment above needs to be updated.